lib/bootstrap.js

```diff
@@ -73,37 +73,46 @@ async function runCLI(cli, commandIsUsed) {
                 return;
             }
         } catch (err) {
-            if (err.name === 'UNKNOWN_VALUE') {
-                process.cliLogger.error(`Parse Error (unknown argument): ${err.value}`);
-                return;
-            } else if (err.name === 'ALREADY_SET') {
-                const argsMap = {};
-                const keysToDelete = [];
-                process.argv.forEach((arg, idx) => {
-                    const oldMapValue = argsMap[arg];
-                    argsMap[arg] = {
-                        value: process.argv[idx],
-                        pos: idx,
-                    };
-                    // Swap idx of overriden value
-                    if (oldMapValue) {
-                        argsMap[arg].pos = oldMapValue.pos;
-                        keysToDelete.push(idx + 1);
-                    }
-                });
-                // Filter out the value for the overriden key
-                const newArgKeys = Object.keys(argsMap).filter(arg => !keysToDelete.includes(argsMap[arg].pos));
-                // eslint-disable-next-line require-atomic-updates
-                process.argv = newArgKeys;
-                args = cmdArgs(core, { stopAtFirstUnknown: false, partial: true });
+            const errs = {
+                UNKNOWN_VALUE: 'UNKNOWN_VALUE',
+                ALREADY_SET: 'ALREADY_SET',
+            };
+            switch (err.name) {
```

Comment:
@anshumanv I think we should stick to if statements for comparisons as we can't add complex conditions if we needed in future on switch.

Reply:
Makes sense, but this seems more readable and if we need complex conditions we can use conditionals inside the switch that we're working with.

Reply:
@anshumanv It is my personal choice but lets consider a scenario like below having if
```js
if (err.name == errs.UNKNOWN_VALUE and err.name == errs.ALREADY_SET) {/** code **/}
```
is more readable to me as compared to
```js
switch(err.name) {
  case errs.UNKNOWN_VALUE:
     if (err.name == errs.ALREADY_SET) {/** code **/}
     break;
}
```

Reply:
Ah, yes. Seems more readable the way you suggested to me as well. I believe err.name will be a single value and from your example, it depends and say there are 3 params in future, it can lead up to 9-if-else combinations at max. But this is highly data-oriented and if err.name is a single value then at all times then we're cool with this arrangement as well. Let's see though, I'm okay with both approaches and as per the majority. 👍

Reply:
Oops got wrong with the example, but you may got the gist of what I was trying to say :)